pkg/kube/secretcontroller/secretcontroller.go

```diff
@@ -50,13 +50,19 @@ const (
 
 func init() {
 	monitoring.MustRegister(timeouts)
+	monitoring.MustRegister(remoteClustersCount)
 }
 
 var timeouts = monitoring.NewSum(
 	"remote_cluster_sync_timeouts_total",
 	"Number of times remote clusters took too long to sync, causing slow startup that excludes remote clusters.",
 )
 
+var remoteClustersCount = monitoring.NewGauge(
+	"istiod_remote_clusters_total",
```

Comment:
is there any time this is not `istiod_clusters_total{} - 1`?

Reply:
I actually don't know. I was hoping that would be exposed in this review. Do external `istiod`s have a local cluster ?

Reply:
There is a "config cluster", which is the primary cluster that we read other multicluster secrets from, CRDs, etc. That may or may not be the cluster istiod actually runs - it could run on a raw VM for instances. I _think_ we can just count the number of clusters we have (istiod_clusters_total) and that should be sufficient?

@nmittler to keep me honest..

Reply:
i've updated to reduce the single gauge. hopefully, this covers the basics. it does not tell us if there is an error setting up mgmt of the remote clusters, etc., however.